docs/editing-docs.rst

```diff
@@ -57,8 +57,8 @@ behind choosing what information to host where.
 Bio-Formats documentation
 -------------------------
 
-Hosted at `<https://docs.openmicroscopy.org/bio-formats/>`_\{{version}} (plus
-latest redirect - :bf_doc:`docs.openmicroscopy.org/latest/bio-formats/ <>`)
+Hosted at `<https://bio-formats.readthedocs.io/en/>`_\{{version}} (plus
```

Comment:
Should we drop the latest/version details and just point to https://bio-formats.readthedocs.io/en?